src/main/resources/interface-specification.yml

```
@@ -2519,6 +2702,91 @@ components:
       schema:
         type: string
   schemas:
+    Mail:
+      type: object
+      required:
+        - address
```

I know you know, but 
```suggestion
      required:
        - address
        - description
```

---

The requirements were to have description optional